go/common/version/version.go

```diff
@@ -102,17 +102,17 @@ var Versions = struct {
 
 func parseSemVerStr(s string) Version {
 	split := strings.Split(s, ".")
-	if len(split) != 3 {
-		panic("version: failed to split SemVer")
-	}
 
-	var semVers []uint16
-	for _, v := range split {
-		i, err := strconv.ParseUint(v, 10, 16)
+	var semVers []uint16 = []uint16{0, 0, 0}
+	for i, v := range split {
+		if i >= 3 {
```

Comment:
might also be good to use https://golang.org/pkg/strings/#SplitN

Reply:
minor performance enhancement